Meal_Application.sql

```diff
@@ -0,0 +1,201 @@
+USE Meal_Sharing;
+
+
+-- Creat the data model.
+     -- diagram is here: https://dbdiagram.io/d/5d5bff66ced98361d6ddc18c
+
+
+CREATE TABLE meal (
+	id INT unsigned NOT NULL AUTO_INCREMENT,
+	title VARCHAR(255),
+    description text,
+	location VARCHAR(255),
+	reservation_time DATETIME,
+	max_reservations INT,
+	price decimal,
+	created_date DATE,
+	PRIMARY KEY(id)
+); 
+
+CREATE TABLE reservation (
+	id INT unsigned NOT NULL AUTO_INCREMENT,
+	number_of_guests INT,
+	meal_id INT unsigned,
+	created_date DATE,
+	PRIMARY KEY (id),
+	CONSTRAINT fk_reservation
+	FOREIGN KEY (meal_id)
+		REFERENCES meal(id)
+		ON UPDATE SET NULL
+		ON DELETE SET NULL
+); 
+
+CREATE TABLE review (
+	id INT unsigned NOT NULL AUTO_INCREMENT,
+	title VARCHAR(255),
+	description TEXT,
+	meal_id INT unsigned,
+	stars INT,
+	created_date DATE,
+	PRIMARY KEY (id),
+	CONSTRAINT fk_review
+	FOREIGN KEY (meal_id)
+		REFERENCES meal(id)
+		ON UPDATE SET NULL
+		ON DELETE SET NULL
+); 
+
+
+
+-- Meal Queries :
+
+-- Get all meals:
+SELECT * FROM meal;
+
+-- Add a new meal
+
+INSERT INTO meal (title, description, location, reservation_time, max_reservations, price, created_date)
+VALUES ('Coffee', 'Wonderful Coffee', 'Home', '2020-02-20 10:00:00', '4', '10', '2020-02-14'); 
+
+-- Get a meal with any id, fx 1
+
+SELECT * FROM meal
+WHERE id = 2;
+
```

Comment:
Just for the record
At this point there is only one record in the table and "id=2" returns nothing. But the query is OK.

Reply:
Yeah, I had problem structuring the code for github because at some point we are adding and then deleting or updating. So id's were getting lost at the point when I finished the exercises. I will do as you suggested as soon as possible. Thank you for suggesting and sorry for the inconvenience.

Reply:
You are most welcome and no need to say sorry for this. You have done a great job.